core/service/round.ts

```diff
@@ -0,0 +1,77 @@
+import ResponseEntity from 'shared/model/response';
+import { Delegate } from 'shared/model/delegate';
+
+interface IRoundSum {
+    roundFees: number;
+    roundRewards: number;
+    roundDelegates: Array<string>;
+}
+
+interface Round {
```

Comment:
Round contains a few slots with related generators (delegates), but not single generator and slot